apollo-router/src/plugins/traffic_shaping/deduplication.rs

```diff
@@ -191,8 +229,17 @@ where
 
         if request.operation_kind == OperationKind::Query {
             let wait_map = self.wait_map.clone();
-
-            Box::pin(async move { Self::dedup(service, wait_map, request).await })
+            let deduplicate_query_ignored_headers = self.deduplicate_query_ignored_headers.clone();
```

Comment:
Let's try to not clone it

Reply:
I am cloning here because...

1. The other items are being cloned to pass in due to the `move` below (I believe)
2. If I try to pass a borrowed value, I get into lifetime errors (yuck)